packages/next/build/webpack/loaders/utils.ts

```diff
@@ -1,6 +1,6 @@
 const defaultJsFileExtensions = ['js', 'mjs', 'jsx', 'ts', 'tsx', 'json']
 const imageExtensions = ['jpg', 'jpeg', 'png', 'webp', 'avif']
-const nextClientComponents = ['link', 'image', 'script']
+const nextClientComponents = ['link', 'image', 'head', 'script']
```

Comment:
We should also add it in both loaders right?

Reply:
This will effect both loaders, I grouped them together into one place in #36135